pyobs/modules/telescope/basetelescope.py

```diff
@@ -308,5 +310,12 @@ async def _update_celestial_headers(self) -> None:
             "SUNDIST": (None if sun_dist is None else float(sun_dist.degree), "Solar Distance from Target"),
         }
 
+    def _get_derotator_offset_from_header(self, hdr, obstime):
```

Comment:
Can we rename this to `_calculate_derotator_position_from_header`?

Reply:
Or better: `_calculate_derotator_position` and don't use the header. The telescope itself knows best, where it's pointing...